tests/files/etc/hooks/exit_hook.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash -l
+
+dos2uni /etc/ansible-deploy/ansible-deploy.yaml
```

Comment:
not very bad.. but maybe it should be done to ansible-deploy.yaml before pushing to repo? :)